src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -52,6 +52,10 @@
     <Compile Include="$(CommonPath)\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
+    <ProjectReference Include="..\..\System.Net.Http\pkg\System.Net.Http.pkgproj">
```

Comment:
The System.Net.Http implementation has changed, and I want this implementation of System.Net.Requests to consume the latest implementation which has not yet been shipped. Should I be referencing the src csproj here then?